### DISCUSSION TITLE
Unable to change controls position

### DISCUSSION THREAD

**Discussion by peter099** [Swordfish90/Lemuroid#586] [Q&A]
I want to move movement controls from the left side to the right side but i am not able to. I tried to drag them to the right side but it is not working. Pinching works because it changes the size of the controls, but dragging doesn't seem to work. Does dragging controls working for anyone?

**Comment by cyruzwings** [comment]
lemuroid use [RadialGamePad](https://github.com/Swordfish90/RadialGamePad) as mention by @Swordfish90 in his [blog ](https://swordfish90.github.io/lemuroid-1-5-0.html), this control can not move like you want to do (switch control pad to the right side)
